src/main/java/com/spotinst/sdkjava/model/bl/azure/statefulNode/LaunchSpecCustomConfiguration.java

```diff
@@ -0,0 +1,89 @@
+package com.spotinst.sdkjava.model.bl.azure.statefulNode;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.util.HashSet;
+import java.util.Set;
+
+public class LaunchSpecCustomConfiguration {
```

Comment:
couldn't find where this object is used.

Reply:
this is used, when we use custom image